app/controllers/admin/web_hooks_controller.rb

```diff
@@ -2,9 +2,18 @@ class Admin::WebHooksController < Admin::AdminController
   before_filter :fetch_web_hook, only: %i(show update destroy)
 
   def index
-    # TODO: Pagination
+    limit = params[:limit].to_i
+    limit = 50 if limit <= 0 || limit > 200
```

Comment:
Shall I make it an constant at 50? I can only imagine an API client would try a different number of `limit` so I set an upper bound here